probe_scraper/transform_probes.py

```diff
@@ -182,10 +182,14 @@ def transform(probe_data, node_data, break_by_channel):
             print "  from: " + str({"node": node_id, "version": readable_version})
             for probe_type, probes in probe_data[channel][node_id].iteritems():
                 # Group the probes by the release channel, if requested
+                if probe_type not in result_data:
+                    result_data[probe_type] = 1
+                else:
+                    result_data[probe_type] +=1
                 extract_node_data(node_id, channel, probe_type, probes, result_data,
-                                  readable_version, break_by_channel)
-
-    return result_data
+                                 readable_version, break_by_channel)
+        new_data[channel] = result_data
```

Comment:
This is not needed if we return both the data structures.

Reply:
But it's giving Error at runner.py "'tuple' object does not support item assignment"
I thing is better idea to make one before returning it.

Reply:
You probably need to modify the call site in order to adjust for the tuple being returned. Let's use two objects, please.

Reply:
![screenshot 6](https://user-images.githubusercontent.com/30230441/44858267-3ab45e00-ac8f-11e8-83f2-619f15a8f051.png)
![screenshot 7 _li](https://user-images.githubusercontent.com/30230441/44858271-3d16b800-ac8f-11e8-80ff-3434e5c7572e.jpg)

I know you want but first you see it.

Difference is that if will return  whole probe_counts after calculating as you can see 2 picture(I have mark 1 and 2 note: do not consider the dictionary just see values in last). The whole probe_counts will calculate all scalar,histogram,event sum and return it. 
If we will store in on dictionary is better(You can see 2 picture I have mark 2). It seems better than first.
What I understand I have said now you can guide?
And you can change in value in both histograms.

Reply:
I'm not entirely sure I understand what you mean, but what we want is to return data from the function and then find a way to present it: that shouldn't be a problem, regardless of how many return values we have.